### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by ISeeTWizard** [grimkor/DeckyFileServer#3]
Hi
I’m using your plugin not very often but was happy that it exists as it makes live easier to transfert data to my steam deck
Unfortunately now it doesn’t work anymore
I can’t start the server anymore and also when going to the settings nothing is saved
Anyone else having that issue?

I also uninstalled and reinstalled the plugin etc but didn’t help

**Comment by grimkor** [comment]
> 

Hey so I spotted the problem yesterday and put in a fix [here](https://github.com/grimkor/DeckyFileServer/commit/8eaf7b2b0f70e299a9049c7da40d039700848d64) and I've put up a [PR with Decky](https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/422) to update it on their store. Once this is merged the update should be in the store to download.

The source of the problem is a change in Python, looks like maybe the steam deck got bumped up a version as I used it not too long ago myself!

Until this PR is merged and the update is then on the store my only recommendation is to stop using it as it cannot be resolved.
